### PR TITLE
Remove orb references: hello-world, hello-build, and welcome

### PR DESCRIPTION
**Pull request by kbatuigas**
# Description
Remove references to out of date orbs, and replace with supported orbs where appropriate. 

Mostly uses the `node` orb as replacement.

This PR also removes some outdated screenshots from Your First Green Build -- as of now, we are not replacing the screenshots but we may revisit at a later date if we deem them helpful for this type of docs content.

# Reasons
https://circleci.atlassian.net/jira/software/projects/DOCSTEAM/boards/412?selectedIssue=DOCSTEAM-6

### CODE REVIEW

**Review by thomasr8 — APPROVED** [review]
This looks really good @kbatuigas 👍

File: jekyll/_cci2/configuration-reference.md [hunk 1]
```diff
@@ -61,18 +61,21 @@ executors | N | Map | A map of strings to executor definitions. See the [Executo
 commands | N | Map | A map of command names to command definitions. See the [Commands]({{ site.baseurl }}/2.0/configuration-reference/#commands-requires-version-21) section below.
 {: class="table table-striped"}
 
-The following example calls an Orb named `hello-build` that exists in the certified `circleci` namespace.
+The following example calls an Orb named `node` that exists in the certified `circleci` namespace.
```
Comment (rosieyohannan) [inline jekyll/_cci2/configuration-reference.md:64]:
```suggestion
The following example calls an orb named `node` that exists in the certified `circleci` namespace.
```

Comment (rosieyohannan) [reply jekyll/_cci2/configuration-reference.md:64]: This was already there so not your work but FYI we don't generally capitalise feature names :-)

File: jekyll/_cci2/hello-world.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ This example adds a job called `build` that spins up a container running a [pre-
 
 1. Create a directory called `.circleci` in the root directory of your local GitHub or Bitbucket code repository.
 
-1. Create a [`config.yml`]({{ site.baseurl }}/2.0/configuration-reference/) file with the following lines (if you are using CircleCI server v2.x, use `version: 2.0` configuration):
+2. Create a [`config.yml`]({{ site.baseurl }}/2.0/configuration-reference/) file with the following lines (if you are using CircleCI Server v2.x, use `version: 2.0` configuration):
```
Comment (rosieyohannan) [inline jekyll/_cci2/hello-world.md:23]:
```suggestion
2. Create a [`config.yml`]({{ site.baseurl }}/2.0/configuration-reference/) file with the following lines (if you are using CircleCI server v2.x, use `version: 2.0` configuration):
```

Comment (rosieyohannan) [reply jekyll/_cci2/hello-world.md:23]: same for server ^^

**Review by rosieyohannan — COMMENTED** [review]
This looks great Kat, thank you! 🎉  Just a couple of capitalisation changes

**Review by rosieyohannan — APPROVED** [review]

**Pull request merged**